Neos.Flow/Configuration/Settings.Security.yaml

```
@@ -98,6 +98,10 @@ Neos:
 
       cryptography:
 
+        # A private, unique key used for encryption tasks. Normally 40 characters long and received from a persistent
+        # filesystem cache. If set to a non-empty string, the cache is not involved anymore.
+        encryptionKey: ''
```

Wouldn't it make sence to set this to `null` instead? Would clearly say, that this property is not set.
```suggestion
        encryptionKey: null
```

---

I digress – the schema says it's a required string, so either make it an empty string by default or adjust the schema. 😎

---

Yeah, sure. But the schema is part of this PR, so we are open for both solutions, right? So question is, what do we expect the value for an not-set encryption key?

---

Something empty… 🙃 Either way is fine.